packages/suite/src/views/wallet/send/components/Outputs/components/Amount/components/Fiat.tsx

```diff
@@ -162,46 +191,22 @@ export const Fiat = ({ output, outputId }: FiatProps) => {
                 isClean
                 data-test={currencyInputName}
                 onChange={(selected: CurrencyOption) => {
-                    // propagate changes to FormState
-                    onChange(selected);
-                    // calculate Amount value
-                    const rate = getFiatRate(fiatRates, selected.value);
-                    const amountValue = getDefaultValue(amountInputName, '');
-
-                    const formattedAmount = new BigNumber(
-                        shouldSendInSats
-                            ? formatAmount(amountValue, network.decimals)
-                            : amountValue,
+                    dispatch(
+                        updateFiatRatesThunk({
+                            ticker: { symbol: network.symbol as NetworkSymbol },
+                            localCurrency: selected.value as FiatCurrencyCode,
+                            rateType: 'current',
+                            // TODO: would be nice to update rates regularly, what timestamp should i put here?
+                            lastSuccessfulFetchTimestamp: 0,
```

Comment:
why not to put Date.now() here?

Reply:
e2bfe2357385caf2414cbd377a59129e6259efad